docs/content/contributing.md

```diff
@@ -125,3 +125,27 @@ to the project. Please verify that you have 2FA enabled on both accounts.
 1. Run `script/release` and follow the instructions.
 1. Once your release PR has been approved and merged, run `script/publish`. You may be prompted to log into your rubygem and npm account.
 1. Lastly, draft a new release from the [releases page](https://github.com/primer/view_components/releases). The tag version should be updated to the newest version. The description should be updated to the relevant CHANGELOG descriptions. Press the `Publish release` button and you're good to go!
+
+## Writing changes in CHANGELOG.MD
+
+This file holds all the changes made in previous versions of Primer View Components and the ones coming to the next version.
```

Comment:
What do you think about putting these guidelines in an HTML comment at the top of the changelog?

Reply:
Better seeing them twice than never! So yeah, good point :)

As an HTML comment on the Markdown file itself right?

Reply:
Right!

I think we could probably remove these guidelines from here at that point.

Reply:
Yeah, I removed it from the contributing doc to not have to maintain it in two different places.